test/unit/location.d

```diff
@@ -0,0 +1,35 @@
+module location;
+
+import dmd.lexer;
+import dmd.globals : Loc;
+
+unittest
```

Comment:
All unit test blocks in the `test/unit` directory should have one or multiple UDAs attached to the unit test block containing a string which describes what the test is testing. This does three things:

* Adds a description of what the test is actually testing
* The unit test runner will output this description if the test failed
* It's possible to filter the tests based on this description. This allows to select individual unit test blocks to be executed, instead of having to execute all tests in a file or the whole test suite